_posts/2020-10-10-temperature-water-content-and-polishing-effects-on-koji.md

```diff
@@ -43,17 +43,19 @@ As an aside, this was the experimental set-up:
 ![Figure 1](/assets/images/temp_effect/Figure 1.png){:.responsive .center}
 *The experimental setup - see Ref. 2*
 
-*Callout:* This setup is a sort of respirometer - I'm not sure of the exact name. Oxygen is produced by electrolysis, which builds up pressure in the system, pushing one side of the mercury manometer down. With enough pressure, the mercury level switch opens, stopping the generation of oxygen. The koji continues consuming oxygen, and one can measure total oxygen consumption by integrating the total current consumed over time. Finally, waste carbon dioxide is absorbed by the NaOH solution.
+<div class="callout">
+💡 This setup is a sort of respirometer - I'm not sure of the exact name. Oxygen is produced by electrolysis, which builds up pressure in the system, pushing one side of the mercury manometer down. With enough pressure, the mercury level switch opens, stopping the generation of oxygen. The koji continues consuming oxygen, and one can measure total oxygen consumption by integrating the total current consumed over time. Finally, waste carbon dioxide is absorbed by the NaOH solution.
+</div>
 
 ### Water content
 
 Another parameter they tested is the **steamed rice water absorption rate**, which is defined as follows:
 
 $$
-\text{Steamed rice water absorption rate (%)} = \frac{\text{Steamed rice weight} - \text{White rice weight}}{\text{White rice weight} }   \times 100\%
+\text{Steamed rice water absorption rate (%)} = \frac{\text{Steamed rice weight} - \text{Dry rice weight}}{\text{Dry rice weight} }   \times 100\%
 $$
 
-Note that this is NOT the same as total water content, since the dry white rice also contains a small amount of water (around 15% by weight). When varying the steamed rice water absorption rate from 24 - 51%, this is what they found:
+Note that this is NOT the same as total water content, since the dry white rice also contains a small amount of water (around 15% by weight). The term "dry rice weight" here is a bit of a misnomer - it's the weight of uncooked, unsoaked rice. When varying the steamed rice water absorption rate from 24 - 51%, this is what they found:
 
```

Comment:
OK - agreed "dry rice weight" is more accurate and added a clarification. I realized that "Uncooked rice" is a really bad choice cause some people will interpret it as soaked, but not steamed.